### PR TITLE
Improve grammar and formatting in README

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,31 +12,31 @@ Add a step like this to your workflow:
...
+    # Whether to use the --force option on `git add`, in order to bypass eventual gitignores
     # Default: false
     force: true
 
     # The message for the commit
     # Default: 'Commit from GitHub Actions'
     message: 'Your commit message'
 
-    #  The arguments for the git rm command (see the paragraph below for more info)
+    #  The arguments for the `git rm` command (see the paragraph below for more info)
```
Comment (EndBug) [inline README.md:39]: Why did you put the backticks here? If this was plain text that would correctly format it as code, but here it's not...

Comment (jsoref) [reply README.md:39]: Grammarly was confused by the sentence:

> The arguments for the git add command (see the paragraph below for more info)

It parsed add as binding to command. My initial fix was changing it to git-add which made Grammarly happy, but I worried someone would complain that git-add isn't a command.

Comment (EndBug) [reply README.md:39]: Hm, I see. The only alternative would be using normal quotes but that wouldn't be much better. I think it's better to keep the backticks then.

**Comment by EndBug** [comment]
Merged! Thanks again for the PR

**Comment by EndBug** [comment]
@all-contributors please add @jsoref for docs

**Pull request merged**